provider/resources.go

```diff
@@ -190,7 +190,6 @@ func Provider() tfbridge.ProviderInfo {
 			"linode_lke_cluster":           {Tok: makeResource(mainMod, "LkeCluster")},
 			"linode_firewall":              {Tok: makeResource(mainMod, "Firewall")},
 			"linode_object_storage_object": {Tok: makeResource(mainMod, "ObjectStorageObject")},
-			"linode_vlan":                  {Tok: makeResource(mainMod, "Vlan")},
```

Comment:
What happened here?

Reply:
This was removed in a previous release - marked as `Removed` in the tF schema. In this release, they just cleaned it up - it's why no files have been removed to go with the change